scripts/compact-junit.py

```diff
@@ -0,0 +1,17 @@
+import sys
+
+from junitparser import JUnitXml, Failure
+
+
+def main():
+    junit_xml_file = sys.argv[1]
+    suite = JUnitXml.fromfile(junit_xml_file)
+    for case in suite:
+        if not isinstance(case.result, Failure):
+            case.system_out = ''
+            case.system_err = ''
+    suite.write()
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
This change is unrelated to "enable parallel testing". Please move this (and the related changes) into a separate commit.